src/app/modals/about-modal.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="about-modal-pf">
+  <div class="modal-header">
+    <button type="button" class="close" (click)="close($event)" aria-hidden="true">
+      <span class="pficon pficon-close"></span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <h1 *ngIf="config.title">{{config.title}}</h1>
+    <div *ngIf="config.productInfo && config.productInfo.length > 0" class="product-versions-pf">
```

Comment:
The HTML looks nice. A good test would be to try your example without providing a config. Although the modal may look odd, there should be no errors in the browser console.